<launch>
  <include file="$(find pozyx_simulation)/launch/uwb_anchors_set.launch" />
  
  <node pkg="pozyx_simulation" name="pozyx_simulation"  type="uwb_simulation.py" output="screen">
        <param name="modelstate_index" value="2" />
  </node>
</launch>
